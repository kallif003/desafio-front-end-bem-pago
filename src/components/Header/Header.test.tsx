import * as React from "react"
import { render } from "@testing-library/react"

import { Header, HeaderTitle } from "./index"

describe("Header components", () => {
	it("Renders Header properly", () => {
		const { container } = render(<Header>Test</Header>)
		expect(container.firstChild).toMatchSnapshot()
	})
})

describe("Header components", () => {
	it("Renders HeaderTitle properly", () => {
		const { container } = render(<HeaderTitle>Test</HeaderTitle>)
		expect(container.firstChild).toMatchSnapshot()
	})
})
