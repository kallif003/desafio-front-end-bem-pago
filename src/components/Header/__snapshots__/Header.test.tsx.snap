// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header components Renders Header properly 1`] = `
<header
  class="sc-bdvvtL kGvRYS"
>
  Test
</header>
`;

exports[`Header components Renders HeaderTitle properly 1`] = `
<div
  class="sc-gsDKAQ iOVEzF"
>
  Test
</div>
`;
